sdk/monitor/opentelemetry-exporter/src/platform/nodejs/utils/httpUtils.ts

```diff
@@ -0,0 +1,80 @@
+import * as http from "http";
```

Comment:
lots of repeated stuff from `core-http`, I think eventually it'd be good to move this over to use the common core